### ISSUE TITLE
ng update gives error when dependency is local only

### ISSUE THREAD

**Issue by hodlbank** [angular/angular-cli#10614]
### Versions
```
Angular CLI: 6.0.0-rc.10
Node: 8.11.1
OS: linux x64
Angular: 6.0.0-rc.6
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.5.13
@angular-devkit/core         0.5.13
@angular-devkit/schematics   0.5.13
@angular/cdk                 6.0.0-rc.14
@angular/cli                 6.0.0-rc.10
@angular/material            6.0.0-rc.14
@schematics/angular          0.5.13
@schematics/update           0.5.13
rxjs                         6.0.0
typescript                   2.8.3
```

### Repro steps

* have Angular 5.x project with local-only dependencies, i.e. dependency added into package.json but does not exist in the npmjs.org repository .
* run `ng update @angular/cli` in order to convert .angular-cli.json into proper .angular.json

### Observed behavior
```
> ng update @angular/cli --migrate-only --from=1.7.4
Registry returned 404 for GET on https://registry.npmjs.org/some-private-package
```

### Desired behavior
I expect that ng update will just convert my .angular-cli.json into proper .angular.json without digging into my dependencies.

**Comment by james-tonicanalytics** [comment]
I am also experiencing this issue.

**Comment by james-tonicanalytics** [comment]
The temporary solution that stopped the error occurring was to delete the local dependency from the package.json file and then run ng update @angular/cli. Once the .angular-cli.json was migrated to angular.json

**Comment by jycouet** [comment]
Hello, 
I have the same issue!

What is really strange is that I have 2 local packages... And when I removed one, it's working!

**Comment by scottleedavis** [comment]
I have this problem too.  Also I have the same problem with a github/gitlab reference, see #10642

**Comment by thorsten** [comment]
Same issue here.

**Comment by Al-Sharif** [comment]
Same here

**Comment by daniloff200** [comment]
hey @hansl, @mgechev 
I think, this issue can be closed, cause #13167 was merged

**Comment by mgechev** [comment]
Please, open a new issue if this is still reproducible with the latest CLI, following the template and providing a minimal demo.
